meta-balena-common/recipes-devtools/go/go-crosssdk.inc

```diff
@@ -1,7 +1,7 @@
 inherit crosssdk
 
 DEPENDS = "go-native virtual/${TARGET_PREFIX}gcc-crosssdk virtual/nativesdk-${TARGET_PREFIX}compilerlibs virtual/${TARGET_PREFIX}binutils-crosssdk"
-PN = "go-crosssdk-${TARGET_ARCH}"
+PN = "go-crosssdk-${SDK_SYS}"
```

Comment:
Here the include is used in recipes that run in the target SDK and outputs code for the SDK machine. I would like to know why `TARGET_ARCH` was used originally instead of `SDK_ARCH`, but maybe `SDK_ARCH` is a better option than `SDK_SYS`.

Reply:
And this comes from https://www.openembedded.org/pipermail/openembedded-core/2019-July/284876.html